clang/docs/ReleaseNotes.rst

```diff
@@ -300,6 +300,11 @@ Attribute Changes in Clang
   to reduce the size of the destroy functions for coroutines which are known to
   be destroyed after having reached the final suspend point.
 
+- Clang now introduced ``[[clang::coro_return_type]]`` and ``[[clang::coro_wrapper]]``
+  attributes. A function returning a type marked with ``[[clang::coro_return_type]]``
+  should be a coroutine. A non-coroutine function marked with ``[[clang::coro_wrapper]]``
+  is still allowed to return the such a type.
```

Comment:
```suggestion
  is still allowed to return the such a type. This is helpful for analyzers to recognize coroutines from the function signatures.
```

nit. Not required. But I feel it reads better if I can understand the intention here.